src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArr = [];
```

Comment:
use better name

and `arr`/`obj` suffix are not needed, I can take this information directly from the code